shared/src/commonMain/kotlin/io.newm.shared/di/Koin.kt

```diff
@@ -90,7 +90,6 @@ fun commonModule(enableNetworkLogs: Boolean) = module {
     single { createJson() }
     // Internal Configurations
     single<NewmSharedBuildConfig> { NewmSharedBuildConfigImpl() }
-    single { TokenManager(get()) }
```

Comment:
The token manager needs to remain in the KMP layer, as iOS also uses the same interface. With different implementations, curious why did we need to move the injection out of the share module?